docs/building-a-form/common-definitions.md

```diff
@@ -153,7 +142,7 @@ The common date field, excluding day field, with basic validation.
 - `schema`: No
 - Function or object: Function
```

Comment:
Probably best for a second PR, but each one that says it's a function should specify what arguments it takes.

Reply:
That seems important, yes. If you have time, could you open a PR to add those?

Reply:
yeah, I'll do that today so we have it before the hackfest!